packages/components/tag/src/types.ts

```diff
@@ -23,3 +23,27 @@ export type TagProps = ExtractInnerPropTypes<typeof tagProps>
 export type TagPublicProps = ExtractPublicPropTypes<typeof tagProps>
 export type TagComponent = DefineComponent<Omit<HTMLAttributes, keyof TagPublicProps> & TagPublicProps>
 export type TagInstance = InstanceType<DefineComponent<TagProps>>
+
+export interface TagData extends Omit<TagProps, 'shape'> {
+  key: VKey
+  text: string
```

Comment:
```suggestion
  key?: VKey
  label?: string
```
text => label 吧，跟其他地方保持一致。
然后后改成非必填吧， key 不存在的时候使用 index